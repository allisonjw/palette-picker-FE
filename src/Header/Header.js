import React from 'react';
import './Header.scss';

const Header = () => {
    return(
        <header className="header">
            <h1>WELCOME TO PALETTE PICKER</h1>
        </header>
    )
}

export default Header;

