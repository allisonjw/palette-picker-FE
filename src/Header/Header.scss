.header {
    background-color: cadetblue;
    display: flex;
    justify-content: center;
    height: 100px;
}

h1 {
    color: white;
    text-align: center;
}