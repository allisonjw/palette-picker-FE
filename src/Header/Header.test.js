import React from 'react';
import { render } from '@testing-library/react';
import Header from './Header';
import { shallow } from 'enzyme';

describe('Header', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Header />)
  });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});