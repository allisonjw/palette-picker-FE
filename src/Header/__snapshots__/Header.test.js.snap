// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match the snapshot 1`] = `
<header
  className="header"
>
  <h1>
    WELCOME TO PALETTE PICKER
  </h1>
</header>
`;
